### DISCUSSION TITLE
why is "Do not display names in feedback" now active by default?

### DISCUSSION THREAD

**Discussion by marcup** [microsoft/vsts-extension-retrospectives#472] [General]
I believe the previous behavior was that feedback was NOT anonymous when creating a new retrospective.
Did you switch that default behavior on purpose? and if so what's the reasoning behind it?
In my opinion retrospectives should encourage transparency, so the less anonymous feedback the better. therefor I would prefer the option of anonymous feedback to be disabled by default.

**Comment by polatengin** [comment]
@dstoddar do you want to chime in and explain the reasoning behind the change?

**Comment by dstoddar** [comment]
@marcup ,
Thank you for raising this for discussion! There are a couple of reasons for doing this. The primary reason is that when going deep on analyzing any of the new team assessment questions it must be anonymous until the data indicates that the team is 100% psychologically safe. Otherwise, individuals that don't feel safe will never share what needs to be shared in order to make improvements. This, by default, would make them feel unsafe because their name would be associated with the feedback about why it doesn't feel safe. Hopefully this makes sense. 
The 2nd reason is because we recently added the "Discuss and Act" button enabling the creation of a board specific to improving the biggest area of opportunity. This creation process must create an anonymous board (because of what I mentioned above), so we turned on "Don't display names" in feedback by default.
You can always uncheck this box in the creation of retrospectives if you choose to. I'd be curious based on what I've shared what your thoughts are.

**Reply by JonSomNor** [reply]
The problem we're facing is that it is no longer possible to uncheck this box. Not even when creating a new retrospective...

**Reply by dstoddar** [reply]
@JonSomNor , can you please provide repro steps? I've just tested on our end and was able to uncheck the box. @enallipogu for visibililty

**Reply by JonSomNor** [reply]
@dstoddar Thanks for responding so quickly!

Maybe I'm just bad at checking boxes, but after some more testing I realized that I just have to be precise and patient. It seems like I have to click well inside the checkbox for the event to be registered. Also, there's a delay from when the click occurs to the control changes value.

So I guess it works as it's supposed to after all...  ;-)

**Comment by dphanson63** [comment]
Defaults are now "sticky" -- they will default to whatever you used last when creating a new board.
